static/styles/portico/landing_page.css

```diff
@@ -3863,3 +3863,17 @@ nav {
     font-size: 13px;
     padding-left: 10px;
 }
+
+.carousel-control.left:hover #prev {
```

Comment:
Your CSS code shouldn't be here it should inside the `.tour .carousel-control` block (on line `1553`).